app/helpers/application_helper/toolbar_chooser.rb

```diff
@@ -385,7 +385,7 @@ def center_toolbar_filename_classic
     to_display = %w[availability_zones cloud_networks cloud_object_store_containers cloud_subnets configured_systems
                     cloud_tenants cloud_volumes ems_clusters flavors floating_ips host_aggregates hosts
                     network_ports network_routers network_services orchestration_stacks resource_pools
-                    security_groups security_policies security_policy_rules storages physical_storages]
+                    security_groups security_policies security_policy_rules storages physical_storages host_initiators]
```

Comment:
you can add `toolbar :host_initiator,:host_initiators` at the end of controller

Reply:
done